<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tela_cadastro_cliente</class>
 <widget class="QMainWindow" name="tela_cadastro_cliente">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>238</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cadastro de Clientes</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_salvar">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Salvar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cancelar">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_novo">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Novo</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_observacoes">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>128</y>
      <width>641</width>
      <height>51</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Observações sobre o Orçamento</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_telefone_cliente">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Telefone para contato</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_data">
    <property name="geometry">
     <rect>
      <x>557</x>
      <y>28</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Data:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_bairro_cliente">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>80</y>
      <width>130</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Bairro da Obra</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_nome_cliente">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>90</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nome do Cliente:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_cidade_cliente">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>60</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cidade:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_numero_cliente">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>45</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Numero:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_vendedores">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Vendedor Responsável</string>
    </property>
    <property name="placeholderText">
     <string>Selecione o vendedor</string>
    </property>
    <item>
     <property name="text">
      <string>Selecione o vendedor</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="lineEdit_data">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>26</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>10</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>250</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Data</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>21/05/2020</string>
    </property>
    <property name="maxLength">
     <number>10</number>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_bairro_cliente">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>60</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Bairro:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_nome_cliente">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>190</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Nome do Cliente</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_endereco_cliente">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>190</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Endereço da Obra ou do Cliente</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_observacoes">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>108</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Observações:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_telefone_cliente">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>50</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Telefone:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_cidade_cliente">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>80</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Cidade da Obra</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_vendedor">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Vendedor Responsável:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_endereco_cliente">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Endereço:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_numero_cliente">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>80</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Numero da Obra</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_anterior">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Anterior</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_proximo">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Proximo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_editar">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Editar</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_nome_cliente</tabstop>
  <tabstop>lineEdit_telefone_cliente</tabstop>
  <tabstop>lineEdit_endereco_cliente</tabstop>
  <tabstop>lineEdit_numero_cliente</tabstop>
  <tabstop>lineEdit_bairro_cliente</tabstop>
  <tabstop>lineEdit_cidade_cliente</tabstop>
  <tabstop>textEdit_observacoes</tabstop>
  <tabstop>comboBox_vendedores</tabstop>
  <tabstop>pushButton_salvar</tabstop>
  <tabstop>pushButton_editar</tabstop>
  <tabstop>pushButton_novo</tabstop>
  <tabstop>pushButton_anterior</tabstop>
  <tabstop>pushButton_proximo</tabstop>
  <tabstop>pushButton_cancelar</tabstop>
  <tabstop>lineEdit_data</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
